File: CS/12_Tables/SetBordersForNewlyTables.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Presentation;

namespace SetBordersForNewlyTables
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Create a PPT document
            Presentation presentation = new Presentation();

            //Set the table width and height for each table cell.
            double[] tableWidth = new double[] { 100, 100, 100, 100, 100 };
            double[] tableHeight = new double[] { 20, 20 };

            //Traverse all the border type of the table.
            foreach (TableBorderType item in Enum.GetValues(typeof(TableBorderType)))
            {
              //Add a table to the presentation slide with the setting width and height
                ITable itable = presentation.Slides.Append().Shapes.AppendTable(100, 100, tableWidth, tableHeight);

                //Add some text to the table cell.
                itable.TableRows[0][0].TextFrame.Text = "Row";
                itable.TableRows[1][0].TextFrame.Text = "Column";

                //Set the border type, border width and the border color for the table.
                itable.SetTableBorder(item, 1.5, Color.Red);
            }

            String result = "Result-SetBordersForNewlyTables.pptx";

            //Save to file.
            presentation.SaveToFile(result, FileFormat.Pptx2013);

            //Launch the PowerPoint file.
            PptDocumentViewer(result);
        }

        private void PptDocumentViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB/12_Tables/SetBordersForNewlyTables.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Presentation

Namespace SetBordersForNewlyTables
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Create a PPT document
			Dim presentation As New Presentation()

			'Set the table width and height for each table cell.
			Dim tableWidth() As Double = { 100, 100, 100, 100, 100 }
			Dim tableHeight() As Double = { 20, 20 }

			'Traverse all the border type of the table.
			For Each item As TableBorderType In System.Enum.GetValues(GetType(TableBorderType))
			  'Add a table to the presentation slide with the setting width and height
				Dim itable As ITable = presentation.Slides.Append().Shapes.AppendTable(100, 100, tableWidth, tableHeight)

				'Add some text to the table cell.
				itable.TableRows(0)(0).TextFrame.Text = "Row"
				itable.TableRows(1)(0).TextFrame.Text = "Column"

				'Set the border type, border width and the border color for the table.
				itable.SetTableBorder(item, 1.5, Color.Red)
			Next item

			Dim result As String = "Result-SetBordersForNewlyTables.pptx"

			'Save to file.
			presentation.SaveToFile(result, FileFormat.Pptx2013)

			'Launch the PowerPoint file.
			PptDocumentViewer(result)
		End Sub

		Private Sub PptDocumentViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace